### PR TITLE
Feature/enable cuda

### PR DESCRIPTION
**Pull request by A7exSchin**

### CODE REVIEW

**Review by JWG-1 — APPROVED** [review]
Works on my system

File: Readme.md [hunk 1]
```diff
@@ -21,21 +21,46 @@
...
+  ``` 
+  For pyTorch 1.4 you need to manually search to correct version [here](https://download.pytorch.org/whl/cu101/torch/), because the official installation methods are broken [[1](https://github.com/pytorch/pytorch/issues/37113)]. For example here we are installing pyTorch 1.4 for the Windows platform on a mobile chip:
+  ```commandline
+    pip install https://download.pytorch.org/whl/cu101/torch-1.4.0-cp37-cp37m-win_amd64.whl
+    ```
+## pyTorch 1.13.1, Python 3.9, CUDA 11.7
+- Install all the requirements from `pip install -r requirements.txt`.
+  - Note: If issues arise installing torch/torch_scatter or when executing the code, try to install them manually, with the following command:
+  ```commandline
+  pip3 install torch==1.13.1 --extra-index-url https://download.pytorch.org/whl/cu117
```
Comment (1Maxnet1) [inline Readme.md]: I would keep it consistent and mention pip instead of pip3

**Comment by 1Maxnet1** [comment]
Everything works for me, approved except for the remaining comment ;)

**Review by RaphaW — APPROVED** [review]
I only checked the old version without cuda since I don't have/work with cuda and it still works for me

**Pull request merged**